ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -568,32 +569,42 @@ private void handleJsonRPCRequest(final RoutingContext routingContext) {
     if (authenticationService.isPresent() && token == null && config.getNoAuthRpcApis().isEmpty()) {
       // no auth token when auth required
       handleJsonRpcUnauthorizedError(routingContext, null, JsonRpcError.UNAUTHORIZED);
-      return;
-    }
-    // Parse json
-    try {
-      final String json = routingContext.getBodyAsString().trim();
-      if (!json.isEmpty() && json.charAt(0) == '{') {
-        final JsonObject requestBodyJsonObject =
-            ContextKey.REQUEST_BODY_AS_JSON_OBJECT.extractFrom(
-                routingContext, () -> new JsonObject(json));
-        AuthenticationUtils.getUser(
-            authenticationService,
-            token,
-            user -> handleJsonSingleRequest(routingContext, requestBodyJsonObject, user));
-      } else {
-        final JsonArray array = new JsonArray(json);
-        if (array.size() < 1) {
-          handleJsonRpcError(routingContext, null, INVALID_REQUEST);
-          return;
+    } else {
+      // Parse json
+      try {
+        final String json = routingContext.getBodyAsString().trim();
+        if (!json.isEmpty() && json.charAt(0) == '{') {
+          final JsonObject requestBodyJsonObject =
+              ContextKey.REQUEST_BODY_AS_JSON_OBJECT.extractFrom(
+                  routingContext, () -> new JsonObject(json));
+          if (authenticationService.isPresent()) {
+            authenticationService
+                .get()
+                .authenticate(
+                    token,
+                    user -> handleJsonSingleRequest(routingContext, requestBodyJsonObject, user));
+          } else {
+            handleJsonSingleRequest(routingContext, requestBodyJsonObject, Optional.empty());
+          }
+
```

Comment:
Stray empty line, can we also add a comment in here explaining what the high level if statement is doing (single request vs batch request based on the first character being { or [ ?